app/javascript/packs/application.js

```diff
@@ -0,0 +1,22 @@
+/* eslint no-console:0 */
+// This file is automatically compiled by Webpack, along with any other files
+// present in this directory. You're encouraged to place your actual application logic in
+// a relevant structure within app/javascript and only use these pack files to reference
+// that code so it'll be compiled.
+
+import 'jquery';
```

Comment:
Do we use jQuery and UJS in the dashboard? I didn't think we did.

Reply:
we do -- the login link has `data-method="post"` on them and without ujs it doesn't POST but GET's instead (which fails)